.github/workflows/tiledb-cloud-py.yaml

```diff
@@ -11,6 +11,10 @@ on:
     # 01:30 UTC = 20:30/21:30 America/New_York = 03:30/04:30 Europe/Athens
     - cron: "30 1 * * *"
 
+env:
+  PYTEST_SPLIT_GROUPS: 6 # How many parallel groups to create for the tests.
+  PYTEST_TEST_DURATIONS_CACHE_KEY: "test_durations"
```

Comment:
Since these variables are specific to the run-tests pipeline, I think it would make sense to define them as part of that job.